modules/admin_manual/pages/release_notes.adoc

```diff
@@ -34,6 +35,119 @@
+
+The most prominent recent improvements are
+
+* Accessibility: Compliance with WCAG 2.1 / BITV 2.0 (currently in the certification process)
+* Theming: Ability to change name, logos, colors and more via config file (see the https://owncloud.dev/clients/web/theming/[documentation] for more information)
+* Performance improvements in many areas
+
+With all these improvements we want to motivate ownCloud service providers to make ownCloud Web available to users, to experience the new technology as well as to provide feedback to further improve it. The ownCloud Web integration app comes as part of the 10.8 complete bundle. Furthermore, ownCloud Web has a release cycle of 3 weeks that is independent of ownCloud Server. This allows to get new versions with features, fixes and improvements regularly via the ownCloud Marketplace and to deploy them with minimal effort.
+
+TIP: ownCloud Web is currently in the status of a Technology Preview. This means that bugs and other undesired behavior are expected. After careful testing, ownCloud Web can be used on production systems. Features are still being added to ownCloud Web and users will need to use the Classic web interface to do certain actions. Please evaluate according to your use case how well the new web interface suits your needs and let us know any feedback that you encounter.
+
+==== Fixed known issues
+
+Both xref:release_notes.adoc#known-issues[known issues from ownCloud Server 10.7] have been fixed. The deployment of ownCloud Web is now more robust and administrators can optionally decide whether ownCloud Links (public and private links) should be provided by the Classic web interface or by ownCloud Web using a https://owncloud.dev/clients/web/deployments/oc10-app/#configure-link-routing[new option in config.php]. 
+
+==== Feedback
+As mentioned, features are still being added to ownCloud Web and the new web interface can't yet cover every use case of the Classic interface. To further shape the new product and to determine the development priorities it is of utmost importance to consider user feedback. We're very grateful for any hints or feedback that you supply via the following channels
+
+- Survey: LINK
```

Comment:
Should `LINK` be replaced by a link or is the term valid

Reply:
Yes, placeholder for now.